packages/handlers/mysql/src/generate-schema.ts

```diff
@@ -0,0 +1,284 @@
+    );
+    return results.map(result => result.TABLE_NAME);
+  }
+
+  private processColumns(tableName: string, columns: ColumnInfo[]) {
+    const createInputFields: GraphQLInputFieldConfigMap = {};
+    const updateInputFields: GraphQLInputFieldConfigMap = {};
+    const outputFields: GraphQLFieldConfigMap<any, any> = {};
+    let primaryKeyFieldName = 'id';
+    for (const column of columns) {
+      const fieldName = column.Field;
+      let columnGraphQLType: GraphQLType;
+      if (column.Key) {
+        columnGraphQLType = GraphQLID;
+        if (column.Key === 'PRI') {
+          primaryKeyFieldName = fieldName;
+        }
+      } else if (column.Type.startsWith('enum(')) {
+        const enumValues = column.Type.replace(`enum('`, '').replace(`')`, '').split(`','`);
+        columnGraphQLType = new GraphQLEnumType({
+          name: pascalCase(fieldName),
+          values: enumValues.reduce(
+            (prev, curr) => ({ ...prev, [curr]: { value: curr } }),
+            {} as GraphQLEnumValueConfigMap
+          ),
+        });
+      } else {
+        const columnTypeName = column.Type.split('(')[0];
+        columnGraphQLType = this.scalarsMap.get(columnTypeName)!;
+        if (!columnGraphQLType) {
+          throw new Error(`Unknown type: ${columnTypeName}`);
+        }
+      }
+      outputFields[fieldName] = {
+        type: column.Null === 'NO' ? new GraphQLNonNull(columnGraphQLType) : columnGraphQLType,
+        description: column.Comment,
+      };
+      createInputFields[fieldName] = {
+        type: column.Null === 'NO' && !column.Default ? new GraphQLNonNull(columnGraphQLType) : columnGraphQLType,
+        description: column.Comment,
+      };
+      updateInputFields[fieldName] = {
+        type: column.Key === 'PRI' ? new GraphQLNonNull(columnGraphQLType) : columnGraphQLType,
+        description: column.Comment,
+      };
+    }
+    const outputType = new GraphQLObjectType({
+      name: pascalCase(tableName),
+      fields: outputFields,
+      description: 'Object Type for table :' + tableName,
+    });
+
+    this.outputTypeMap.set(tableName, outputType);
+
+    const readAllOperationName = camelCase('getAll_' + tableName);
+    this.queryFields[readAllOperationName] = {
+      type: new GraphQLList(outputType),
+      resolve: async (root, args, context, info) => {
+        const fields = Object.keys(graphqlFields(info));
+        return this.query(`SELECT ${fields.join(',')} FROM ${tableName}`);
+      },
+    };
+
+    const readOperationName = camelCase('get_' + tableName + '_By_' + primaryKeyFieldName);
+    this.queryFields[readOperationName] = {
+      type: outputType,
+      args: {
+        [primaryKeyFieldName]: {
+          type: new GraphQLNonNull(GraphQLID),
+        },
+      },
+      resolve: async (root, args, context, info) => {
+        const fields = Object.keys(graphqlFields(info));
+        const result = await this.query(`
+                    SELECT 
```

Comment:
I think we should avoid doing queries this way, we should use parameterized queries. 
I think the NodeJS driver supports it with `?` and passing an array of parameters in order.